code/controllers/subsystem.dm

```diff
@@ -277,8 +277,9 @@
 	var/time = (REALTIMEOFDAY - start_timeofday) / 10
 	var/msg = "Initialized [name] subsystem within [time] second[time == 1 ? "" : "s"]!"
 	if(GLOB.is_debug_server)
-		to_chat(world, span_debug("[msg]"))
-	log_world(msg)
+		message_debug(DBG_INFO, "MC/Initialize", msg)
+	else
+		log_world(msg)
```

Comment:
Don't make these debug spans, I like the way it looks as-is.

Reply:
Unavoidable without manually styling, as this used the old debug span.